legacy/messaging/src/main/resources/schema/jboss-as-messaging_1_4.xsd

```diff
@@ -108,6 +108,13 @@
                   </xs:documentation>
               </xs:annotation>
           </xs:element>
+          <xs:element maxOccurs="1" minOccurs="0" name="override-in-vm-security" type="xs:boolean">
```

Comment:
Not really sure about that. 1.4 parser supports this. it is clearly just bug in schema.